pkg/square/square.go

```diff
@@ -110,6 +217,19 @@ func (s Square) Equals(other Square) bool {
 	return true
 }
 
+// WrappedPFBs returns the wrapped PFBs in a square
+func (s Square) WrappedPFBs() (core.Txs, error) {
+	wpfbShareRange, err := shares.GetShareRangeForNamespace(s, namespace.PayForBlobNamespace)
+	if err != nil {
+		return core.Txs{}, nil
+	}
+	return shares.ParseTxs(s[wpfbShareRange.Start : wpfbShareRange.End+1])
```

Comment:
[question]

```go
// namespace. Ranges here are always end exlusive.
func GetShareRangeForNamespace(shares []Share, ns namespace.Namespace) (Range, error) {
```

so they is the `+1` necessary for `wpfbShareRange.End+1`?

Reply:
Good catch! wasn't even looking there